<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Video">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>790</width>
      <height>450</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox 
{
	border: 1px solid black;
background: white;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="LogoLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>391</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_Backword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play_Pause">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seek_forward">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volume">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_Time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_Time">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_Video_File_Here">
    <property name="title">
     <string>Select Video File Here</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuSelect_Video_File_Here"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
